cs/_pages/technical-details.markdown

```diff
@@ -13,58 +13,81 @@ header:
 toc: true
 ---
 
-This page contains some technical background of TermIt. It describes the architecture of the system, libraries developed by us that it uses, and the data model at its core.
+# Technické detaily
 
+Tato stránka obsahuje další technické pozadí systémy TermIt. Popisuje architekturu systému, knihovny, které jsme vyvinuli a které TermIt používá a datové model použité v základníc verzi TermItu.
```

Comment:
**datový** model **použitý** v **základní** verzi TermItu
(necítíš-li se moravsky)